specs/multispec.yml

```diff
@@ -62,7 +62,7 @@ specs:
        img_name: "{{ spec.org }}/postgresql-{{ spec.short }}"
        pkgs: "postgresql-server postgresql-contrib"
        environment_setup: >-4
-           yum -y install postgresql && \
+          "{ yum -y module enable postgresql:{{ spec.version }} || :; } && \"
```

Comment:
```suggestion
           { yum -y module enable postgresql:{{ spec.version }} || :; } && \
```

Reply:
If you check what the last commit does you will find out that I had exactly this version before. And it failed with syntax error as well.
I am open to suggestions how to fix this, since this does not fail for me locally.